dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/DPPPTController.java

```diff
@@ -108,8 +111,11 @@ public DPPPTController(DPPPTDataService dataService, EtagGeneratorInterface etag
 		if(!this.validateRequest.isFakeRequest(principal, exposeeRequest)) {
 			dataService.upsertExposee(exposee, appSource);
 		} 
+		
+		long after = System.currentTimeMillis();
+		long duration = after - now;
 		try{
-			Thread.sleep(random.nextInt(600));
+			Thread.sleep(this.requestTime - duration);
```

Comment:
sollte evt noch Math.max(this.requestTime - duration, 0) sein? Siehe zb auch https://stackoverflow.com/questions/50269871/thread-sleep-1

Reply:
Unbedingt, sonst gibt es eine Exception wenn der DB insert zu lange dauert!